### DISCUSSION TITLE
Open access for unauthenticated, guests - only for some files.

### DISCUSSION THREAD

**Discussion by nvroot** [coder/code-server#6967] [Q&A] [answered]
Actually running code-server on self-hosted domain with authentication over $SERVICE_PASSWORD, but all files available only for authenticated user, Is there any way to make only some of files open public for everyone ( guests ), like making only index.html https://domain/proxy/3000/index.html - public for everyone, but all other files only authenticated with fast swap access like on/off? Server on extension: Live-Preview

**Comment by code-asher** [comment, marked as answer]
No, currently the auth is all or nothing.  It would be nice to have the ability to make some ports public through the proxy: https://github.com/coder/code-server/issues/6008
